README.md

```diff
@@ -81,6 +81,14 @@ change signature parameters like the algorithm of the signature.
 
 A string which will be used as single key if `keys` is not provided.
 
+##### encryptionKey
+
+If set the cookie will be encrypted using this key.
```

Comment:
I think this description is misleading, especially if folks are going to try and read the cookie in other programming languages. Looking at the code, the value here is not the encryption key, as it is fed into a hash function and that output is used as the key.

Reply:
better now?